TS/treesheets.desktop

```diff
@@ -2,12 +2,13 @@
 Version=1.0
 Type=Application
 Name=TreeSheets
-GenericName=Treesheets
-Comment=A hierarhical spreadsheet / outliner productivity tool.
+GenericName=Spreadsheet
```

Comment:
Why is this changed to Spreadsheet? It isn't a spreadsheet..

Reply:
What would you suggest? I based it on your own description:

> Free Form Data Organizer
> (Hierarchical Spreadsheet)

and

> A "hierarchical spreadsheet" that is a great replacement for spreadsheets, mind mappers, outliners, PIMs, text editors and small databases.

Reply:
either of those is fine..

Reply:
Ok I changed it to Hierarchical Spreadsheet.